src/index.js

```diff
@@ -1,17 +1,28 @@
 import { loaded } from 'document-promises';
 
 customElements.define('iframe-defer', class extends HTMLElement {
+  constructor() {
+    super();
+    this._iframe = document.createElement('iframe');
+  }
+
+  static get observedAttributes() {
+    return ['width', 'height'];
+  }
+
+  attributeChangedCallback(name, oldValue, newValue) {
+    this._iframe.setAttribute(name, newValue);
+  }
+
   connectedCallback() {
     loaded.then(() => {
       if (!this.closest(':root')) return;
 
-      const iframe = document.createElement('iframe');
-
       for (const attr of this.attributes) {
```

Comment:
This loop can be removed, `attributeChangedCallback` will fire on `<defer-iframe>` creation, so it will pass attrs to real `<iframe>`.